setup_new.py

```diff
@@ -0,0 +1,210 @@
+"""
```

Comment:
this name is confusing, is it doing something similar as `setup.py`? Using `new` in a file name is not good practice.

Reply:
To setup a virtual environnement automatically, `conda` is a platform independent solution. We should discuss the goal of this file and if there are not already made, standard ways to do this :)

Reply:
This is designed to be run on a new installation of LabGUI, as it creates a virtual environment, installs all of the requirements on said virtual environment, creates a launcher (batch or bash depending on os), and optionally creates a config file.

I would prefer to use this over conda, as in the past I have had issues with conda (infact I dont even use it anymore). This would give us full control of how the virtual environment is setup for the end-user to work as best as possible with LabGUI.

This script is launched by setup.py, as after creating the virtual environment and creating a launcher, setuptools will copy the launcher to PYTHONPATH and name it LabGUI so that LabGUI can be launched via run/command line/etc. That said, it can also be used independently from setup.py, such as to generate a new virtual environment for another version of python (as it determines the folder of the virtual environment based on the major/minor version of python).

I am completely open to renaming it.

Moreover, I need to test it with Linux before it is ready to be implemented with master. I will try to have this done today.